images/builder/main.go

```diff
@@ -0,0 +1,317 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"math/rand"
+	"os"
+	"os/exec"
+	"path"
+	"strings"
+	"sync"
+	"time"
+
+	yaml "gopkg.in/yaml.v2"
```

Comment:
We should vendor this

Reply:
Bazel currently fetches it with a WORKSPACE rule, so we are pinned to a specific revision.

I opted not to actually vendor it into `vendor/` because that'd mean switching to actually use a dependency manager in this repo, and there are no other places that currently utilise anything outside of the Go stdlib.